Fix enterprise-numbers URL

IANA has changed their URL scheme, and the content at the old URL for
enterprise-numbers switched from text/plain to text/html.

Fix Makefile.am to use the new URL
https://github.com/ipmitool/ipmitool/commit/1edb0e27e44196d1ebe449aba0b9be22d376bcb6

Also fix manpages (no corresponding upstream commit; repository is archived and
no longer accepts contributions)
--- Makefile.am.orig
+++ Makefile.am
@@ -41,7 +41,7 @@ MAINTAINERCLEANFILES = Makefile.in aclocal.m4 configure configure-stamp \
 	$(distdir).tar.gz $(distdir).tar.bz2
 
 SUBDIRS = lib src include doc contrib control
-IANA_PEN = http://www.iana.org/assignments/enterprise-numbers
+IANA_PEN = http://www.iana.org/assignments/enterprise-numbers.txt
 
 dist-hook:
 	cp control/ipmitool.spec $(distdir)
--- doc/ipmievd.8.in.orig
+++ doc/ipmievd.8.in
@@ -221,7 +221,7 @@
 .SH FILES
 .TP
 .B @IANADIR@/enterprise-numbers
-system IANA PEN registry taken from http://www.iana.org/assignments/enterprise-numbers
+system IANA PEN registry taken from http://www.iana.org/assignments/enterprise-numbers.txt
 .TP
 .B ~/@IANAUSERDIR@/enterprise-numbers
 user's override for the system IANA PEN registry, this file if it exists is loaded instead
--- doc/ipmitool.1.in.orig
+++ doc/ipmitool.1.in
@@ -3831,7 +3831,7 @@
 .SH FILES
 .TP
 .B @IANADIR@/enterprise-numbers
-system IANA PEN registry taken from http://www.iana.org/assignments/enterprise-numbers
+system IANA PEN registry taken from http://www.iana.org/assignments/enterprise-numbers.txt
 .TP
 .B ~/@IANAUSERDIR@/enterprise-numbers
 user's override for the system IANA PEN registry, this file if it exists is loaded instead
